ui/src/shared/components/DropdownPlaceholder.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {SFC} from 'react'
```

Comment:
awesome concept, this file

Reply:
inspired by React “suspense” :) I might talk about this at the next lunch and learn

https://www.youtube.com/watch?v=v6iR3Zk4oDY